guides/source/getting_started.md

```diff
@@ -205,7 +205,7 @@ $ rails server
 TIP: If you are using Windows, you have to pass the scripts under the `bin`
 folder directly to the Ruby interpreter e.g. `ruby bin\rails server`.
 
-TIP: Compiling JavaScript asset compression requires you
```

Comment:
This lacks continuity, it says compression here and compilation below. @sharang-d can you check all occurrences for this?